### ISSUE TITLE
Sensitivity problem with combination

### ISSUE THREAD

**Issue by chuckWu1989** [atlassian/react-beautiful-dnd#1809]
Hi, forks. Again, Appreciate with contributors and authors to create so wonderful library for us 🙏

## Question 

Recently, I faced a problem about the sensitivity of combination. It's too sensitive to combine the source into target when the height of target is small. I create a interesting example to explain my scenario 🤪

![image](https://user-images.githubusercontent.com/14174895/78640241-02044e00-78e2-11ea-8f8d-a69093963fa9.png)

In example, the height of row is 24px. According to the logic of rbd, the threshold is 6px ( 1/4 of height). When the bottom of box touched the threshold line, It would be considered as out-boxing condition so that it would trigger reorder impact instead of combination impact. The problem is that 6px is too small. It would be easily judged as reorder.

I had tried to tune the threshold into 1/8, 1/16. I found it could solve the sensitivity problem, but it comes with another problem, the reorder impact would be difficult to trigger.

## Idea

I had two ideas to solve this problem, If there is any advise please join the discussion with us 🤔

First idea is to change the combination judgement condition in rbd. Now, the combination judgement condition is followed by box spacing.

![图片 1](https://user-images.githubusercontent.com/14174895/78642286-0ed67100-78e5-11ea-9047-791c0ecc4da9.png)

When the box spacing touched the threshold, it would trigger the reorder action. I feel it's a little strange. In my experience (not for everyone), I feel the hand position is the position to judge if I touched the threshold.

![图片 2](https://user-images.githubusercontent.com/14174895/78642640-a5a32d80-78e5-11ea-9d67-76edabeb5a89.png)

The reason that I has this feeling is that If I drag the source on the edge of box, It would trigger reorder action in no-common sense position:

![图片 3](https://user-images.githubusercontent.com/14174895/78643145-5c071280-78e6-11ea-95ba-609f4483755e.png)

This is the first idea.

The second idea is about the parameter of combineThresholdDivisor. I think we can consider to release this parameter as a property. The reason is that I think in different size of container, the combineThresholdDivisor could be different. For example, if I have a container with 200px height. According to current combineThresholdDivisor, the threshold would be 50px. I feel it's too large. If we release combineThresholdDivisor as a props, the user could adjust it with their scenario.

## Furthermore

Now I'm modifying this part codes for my project. If you are interested with this idea, I want to contribute 😍. Besides, I created a lovely example about the blog to display sensitivity example. I want to share this example to everyone 🙆‍♂️

![image](https://user-images.githubusercontent.com/14174895/78643812-71307100-78e7-11ea-8b57-0718565d037b.png)

**Comment by maloguertin** [comment]
@chuckWu1989 did you ever find a solution for this?

**Comment by paulkwchoi92** [comment]
How do you tune the threshhold for combining?

**Comment by chuckWu1989** [comment]
@maloguertin Yes, In my issue I mentioned the solution I found. The rbd designed for scrum board scenario originally. Therefore the sensor to detect the combine event is followed design. In order to fit the thin height scenario, I found we can modified the sensor mechanism to detect the drag point instead of edge point (Consider with scrum board scenario, this change should be a option in parameter). Furthermore, as I mentioned In issue, another problem is combineThresholdDivisor. In rbd it's quarter of width/height. In thin height scenario, It's not suitable. The threshold would be too small to trigger the combine event, so I think we could release this parameter for users to define the threshold by themselves.

**Comment by chuckWu1989** [comment]
@maloguertin combineThresholdDivisor is an existed parameter in rbd but it's not released for users. At this moment rbd could not adjust it. Therefore, I advise to release combineThresholdDivisor parameter for users.

**Comment by paulkwchoi92** [comment]
@chuckWu1989 any idea if they would ever be pushing combineThreshholdDivisor to users? or if theres a way to access it?

**Comment by chuckWu1989** [comment]
@paulkwchoi92 Yes, as mentioned in issue. By tuning the combineThresholdDivisor and change the sensor mechanism to detect the drag point instead of edge point, the combine event would be trigger easier in thin height scenario.

Here is a gif to show how the effect If I tuned the combineThresholdDivisor and change the sensor mechanism.

![test1](https://user-images.githubusercontent.com/14174895/114139275-6c126180-9941-11eb-82de-cfc025f3411b.gif)
